packages/core/src/ChatMessage/ChatMessage.tsx

```diff
@@ -79,7 +79,14 @@ function ChatMessage({
       )}
       <Text textStyle='paragraph2'>{message}</Text>
       {footer && (
-        <Absolute bottom='-20px' left='0px' minWidth='300px' width='100%' px={3}>
+        <Absolute
+          bottom='-20px'
+          left={variation === 'outgoing' ? '' : '0px'}
+          right={variation === 'outgoing' ? '0px' : ''}
```

Comment:
An empty string doesn't look to be a valid value for `left`, `right`, ect. ([source](https://developer.mozilla.org/en-US/docs/Web/CSS/left#values)).  

```
variation === 'outgoing' ?
<Absolute
     ...
    right={0}
    px={0}
>
   {footer}
</Absolute> : 
<Absolute
     ...
    left={0}
    px={3}
>
   {footer}
</Absolute> 
```